src/frontend/src/lib/components/receive/ReceiveAddresses.svelte

```diff
@@ -38,167 +39,137 @@
 	import { testnets } from '$lib/derived/testnets.derived';
 	import { i18n } from '$lib/stores/i18n.store';
 	import { modalStore } from '$lib/stores/modal.store';
+	import type { OptionBtcAddress, OptionEthAddress } from '$lib/types/address';
+	import type { Network } from '$lib/types/network';
 	import type { ReceiveQRCode } from '$lib/types/receive';
+	import type { Token } from '$lib/types/token';
 
 	const dispatch = createEventDispatcher();
 
 	const displayQRCode = (details: Omit<Required<ReceiveQRCode>, 'qrCodeAriaLabel'>) =>
 		dispatch('icQRCode', details);
+
+	interface ReceiveAddress {
```

Comment:
There is a cmp name `ReceiveAddress`, maybe `ReceiveAddressProps` to avoid conflicts?